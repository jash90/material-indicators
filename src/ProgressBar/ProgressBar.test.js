import React from 'react';
import ProgressBar from './ProgressBar.js';

import ShallowRenderer from 'react-test-renderer/shallow';
const renderer = new ShallowRenderer();

test('ProgressBar Renders', () => {
  const tree = renderer.render(<ProgressBar />);
  expect(tree).toMatchSnapshot();
});
