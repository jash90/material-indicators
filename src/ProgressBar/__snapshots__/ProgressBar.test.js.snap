// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProgressBar Renders 1`] = `
<Context.Consumer>
  [Function]
</Context.Consumer>
`;
